posts/innovative-snowflake-features-architecture.md

```diff
@@ -0,0 +1,126 @@
+---
+authors:
+- Pooja Krishnan
+tags:
+- Cloud
+- Snowflake
+date: 2019-08-07T10:33:00.000Z
+title: "Innovative Snowflake Features Part 1: Architecture"
+image:
+
+---
+
+Earlier this year, Ippon published an [Introduction to Snowflake](https://blog.ippon.tech/introduction-to-snowflake/) post which provided a Snowflake Primer. I recently attended the Snowflake Partner Bootcamp in Baltimore, a four day instructor led training program, and was given the opportunity to learn even more. In this first of three posts, I am going to do an in-depth examination of Snowflake's unique architecture style and some of the benefits resulting from it. Before we get started, I've included a small section on the different Snowflake Editions as well as a link to the Documentation page for them.
+
+---
+# Snowflake Editions
```

Comment:
Would it be better to maybe move the Snowflake editions section  or added elsewhere? To me, it doesn't go in the flow of the focus of this post i.e. Architecture